Expensify/App
# Firefox TextArea height issue

### Details
Firefox adds height to the TextArea to reserve place for scroll-bars. Because of that, initial heights are different in firefox and chrome. This causes scrollHeight difference and height issue on firefox.

### Fixed Issues

$ https://github.com/Expensify/App/issues/7103

### Tests

1. Go to a conversation
2. In compose textarea verify "Write something..." aligned correctly.
3. Verify empty textarea is 1 line.
4. Verify text with 2+ lines is not scrolling.
5. Edit a previous message.
6. Repeat 3 and 4.

### QA Steps



1. Go to a conversation
2. In compose textarea verify "Write something..." aligned correctly.
3. Verify empty textarea is 1 line.
4. Verify text with 2+ lines is not scrolling.
5. Edit a previous message.
6. Repeat 3 and 4.

### Tested On


### Screenshots


#### Web
https://user-images.githubusercontent.com/1279379/149392964-4e819e5a-78a1-401b-b966-84c96703ebfd.mp4

https://user-images.githubusercontent.com/1279379/149392983-e1e8afd6-5060-44f9-9373-06e6007999e9.mp4


#### Mobile Web


https://user-images.githubusercontent.com/1279379/149393240-f6d37053-ec6f-44ae-a3fa-31ac4e25025a.mp4


https://user-images.githubusercontent.com/1279379/149393250-c9cecaba-1642-4d0d-b43c-1012cf615286.mp4




#### Desktop


https://user-images.githubusercontent.com/1279379/149416979-f58585ef-9a9b-40d7-8221-a734d96310df.mp4


#### iOS


https://user-images.githubusercontent.com/1279379/149416992-676dac0a-8e5a-4e27-b762-3ad68d417e9e.mp4


#### Android


https://user-images.githubusercontent.com/1279379/149393273-f62c4774-6d15-4e34-9eda-525fa4fff4a8.mp4
